src/platform/testing/e2e/cypress/support/commands/expandAccordions.js

```diff
@@ -7,20 +7,12 @@ Cypress.Commands.add('expandAccordions', () => {
 
   cy.get('main').then($main => {
     // Check if va-accordion-item Web Component exists
-    if ($main.find('va-accordion-item').length > 0) {
-      cy.get('va-accordion-item')
+    if ($main.find('va-accordion').length > 0) {
+      cy.get('va-accordion')
```

Comment:
@pdavies88 can you provide some more context here? Would we need to keep the `$main.find()` _and_ the `cy.get()`?

Reply:
@bkjohnson Cypress isn't smart enough to combine conditionals so you have to wrap them instead and the if else conditionals are checked via determining if there is length in the find. So find determines the presence of the element and then the get finds the element. If you don't do the find and just the get and the element doesn't exist on the page being tested with the suite will error and fail. So find ensures that situation does not occur by determining its presence first.